tests/providers/aws/services/iam/lib/policy_test.py

```diff
@@ -89,3 +91,119 @@ def test_is_policy_public(self):
         assert is_policy_public(policy2)
         assert not is_policy_public(policy3)
         assert not is_policy_public(policy4)
+
+    def test_check_full_service_access(self):
+        policy1 = {
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": "s3:*",
+                    "Resource": "*",
+                }
+            ]
+        }
+        policy2 = {
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": "s3:Get*",
+                    "Resource": "*",
+                }
+            ]
+        }
+        policy3 = {
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": "s3:*",
+                    "Resource": "arn:aws:s3:::example_bucket/*",
+                }
+            ]
+        }
+        policy4 = {
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": "s3:*",
+                    "Resource": "arn:aws:s3:::example_bucket",
+                }
+            ]
+        }
+
+        assert check_full_service_access("s3", policy1)
+        assert not check_full_service_access("s3", policy2)
+        assert not check_full_service_access("s3", policy3)
+        assert not check_full_service_access("s3", policy4)
```

Comment:
Could you please create one test case for each policy? It's simpler to work with the tests are failing.